overrides/windows-override.json

```diff
@@ -1,5 +1,16 @@
 {
     "features": {
+        "networkProtection": {
+            "state": "enabled",
+            "features": {
+                "waitlist": {
+                    "state": "disabled"
```

Comment:
Just confirming this should be `disabled` while `waitlistBetaActive` is `enabled`?